### PR TITLE
[G2M] slider - range slider base components with label variant

### PR DESCRIPTION
**Pull request by kc-leung**
- Added range slider base component
- Added range slider label variant
- Added Constant.css for pseudo elements dynamic color styling 
- Added getTailwindConfigColor hook to dynamically get color classes value

### CODE REVIEW

File: src/base-components/range-slider-base/index.js [hunk 1]
```diff
@@ -1,14 +1,22 @@
 import React, { forwardRef, useState, useRef, useCallback, useEffect } from 'react'
 import PropTypes from 'prop-types'
 
+import { getTailwindConfigColor } from '../../hooks/getTailwindConfigColor'
```
Comment (woozyking) [inline src/base-components/range-slider-base/index.js:4]: for file names, please follow `lower-snake-case` style as recommended in https://github.com/EQWorks/common/blob/main/naming.md#code-and-files-associative-naming

**Review by sallkall — APPROVED** [review]
we can revisit the styling when the new design is ready

File: src/components/range-slider-label/range-slider-label.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@import "../../styles/constants.css";
```
Comment (woozyking) [inline src/components/range-slider-label/range-slider-label.css:1]: would this work in the context of other apps without explicit webpack config?

**Review by woozyking — APPROVED** [review]
looking good

File: src/components/range-slider-label/range-slider-label.css (hunk 2 above)
Comment (kc-leung) [reply src/components/range-slider-label/range-slider-label.css:1]: you mean the constants.css file? mmm I think so but probably for webpack app will need css loader I guess...

**Review by hyx131 — APPROVED** [review]
Code looks good! Tried on deployed storybook & couldn't slide the slider tho

File: src/components/range-slider-label/range-slider-label.css (hunk 2 above)
Comment (woozyking) [reply src/components/range-slider-label/range-slider-label.css:1]: the `@import`, but i guess the browser should be able to resolve

**Comment by woozyking** [comment]
> Code looks good! Tried on deployed storybook & couldn't slide the slider tho

on chromatic? i could slide at https://www.chromatic.com/component?appId=60d0cd6c298707003972699d&name=Range+Slider&specName=Base&buildNumber=39

**Comment by hyx131** [comment]
> > Code looks good! Tried on deployed storybook & couldn't slide the slider tho
> 
> on chromatic? i could slide at https://www.chromatic.com/component?appId=60d0cd6c298707003972699d&name=Range+Slider&specName=Base&buildNumber=39

couldn't slide this either, it doesn't work using Chrome, tried on Firefox it works

**Comment by kc-leung** [comment]
> > > Code looks good! Tried on deployed storybook & couldn't slide the slider tho
> > 
> > 
> > on chromatic? i could slide at https://www.chromatic.com/component?appId=60d0cd6c298707003972699d&name=Range+Slider&specName=Base&buildNumber=39
> 
> couldn't slide this either, it doesn't work using Chrome, tried on Firefox it works

mmm cross browser styling issue it seems that I have to separate the pseudo elements to handle them can't have them together. it should be fixed now

**Comment by hyx131** [comment]
> > > > Code looks good! Tried on deployed storybook & couldn't slide the slider tho
> > > 
> > > 
> > > on chromatic? i could slide at https://www.chromatic.com/component?appId=60d0cd6c298707003972699d&name=Range+Slider&specName=Base&buildNumber=39
> > 
> > 
> > couldn't slide this either, it doesn't work using Chrome, tried on Firefox it works
> 
> mmm cross browser styling issue it seems that I have to separate the pseudo elements to handle them can't have them together. it should be fixed now

noice, works now 👍

**Pull request merged**